pod/perlvar.pod

```diff
@@ -25,16 +25,26 @@ to hold data captured by backreferences after a regular expression
 match.
 
 Since Perl v5.6.0, Perl variable names may also be alphanumeric strings
-preceded by a caret.  These must all be written in the form C<${^Foo}>;
-the braces are not optional.  C<${^Foo}> denotes the scalar variable
+preceded by a caret.  These must all be written using the demarcated
+variable form using curly braces such as C<${^Foo}>;
+the braces are B<not> optional.  C<${^Foo}> denotes the scalar variable
 whose name is considered to be a control-C<F> followed by two C<o>'s.
+(See L<perldata/"Demarcated variable names using braces"> for more
+information on this form spelling a variable name or lookup).
```

Comment:
The wording here is weird; I'm not quite sure what it's saying.

Reply:
Changed to:
```
See L<perldata/"Demarcated variable names using braces"> for more
information on this form of spelling a variable name or specifying
access to an element of an array or a hash
```
hopefully that makes more sense!